docs/backend/quantization.ipynb

```diff
@@ -0,0 +1,281 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantization\n",
+    "\n",
+    "`SGLang` support various quantization methods, including online dynamic quantization and offline quantization.\n",
+    "\n",
+    "Please visit [here](https://huggingface.co/collections/neuralmagic) for some popular quantized LLMs on huggingface.\n",
```

Comment:
This is pretty cool. But some model has official quantization. We'd better say that use the official quantized model, for example, llama xxx. If not, you can refer to  xxx.

Reply:
Official, third party, and do it your self